changelog.rst

```diff
@@ -7,6 +7,46 @@
 Change Log
 **********
 
```

Comment:
Please add a section for the 4.4.1 release. To find the changes, look at PRs merged between July 4th and August 29th, dividing between new features and bug fixes, acknowledging the contributors. It contains mainly fixes and improvements by @Ericgig. 

We can already add the PRs for 4.5.x, looking at those merged after August 29th.